backend/tournesol/suggestions/suggester.py

```diff
@@ -0,0 +1,236 @@
+from django.db.models import Q, QuerySet
+
+from core.models import User
+from tournesol.models import ComparisonCriteriaScore, Entity, Poll
+from tournesol.suggestions.graph import Graph
+from tournesol.suggestions.suggested_user import SuggestedUser as RecommendationUser
+from tournesol.suggestions.suggested_video import SuggestedVideo
+
+
+class Suggester:
+    """
+    An interface class to ask for suggestions of videos to compare
+    """
+    # Entity to specific video structure dictionary
+    _entity_to_video: dict[Entity, SuggestedVideo]  # todo Use only uid as key
```

Comment:
Using uids as keys is an important improvement to reduce the memory usage of this structure. No reason to keep all entities metadata in RAM.

Reply:
Done :)